_data/harnesses/generate-test-data-streams/kafka.yml

```diff
@@ -83,6 +83,23 @@ dev:
           render:
             file: tutorials/generate-test-data-streams/kafka/markup/dev/check-connector-02.adoc
 
+        - action: execute
```

Comment:
this causes dupe content to get rendered:

![image](https://github.com/confluentinc/kafka-tutorials/assets/4550245/1417e179-f0f1-4449-bdaa-42b802a1981d)

Reply:
Fixed